src/main/java/com/project/javarush/services/JSONParser.java

```diff
@@ -0,0 +1,44 @@
+package com.project.javarush.services;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.project.javarush.entity.Answer;
+import com.project.javarush.entity.Question;
+import com.project.javarush.repositories.AnswerRepository;
+import com.project.javarush.repositories.QuestionRepository;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+
+public class JSONParser {
+    private QuestionRepository questionRepository;
+    private AnswerRepository answerRepository;
+
+    public QuestionRepository parseQuestionMap(InputStream jsonQuestionFile){
+
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
do we need a new instance of `ObjectMapper` for each method invocation?

Reply:
The class has been refactored, additional class has been added